backend/src/test/kotlin/org/loculus/backend/controller/submission/SubmitProcessedDataEndpointTest.kt

```diff
@@ -758,6 +760,139 @@ class SubmitProcessedDataEndpointTest(
         assertThat(response.body(), `is`("FileV2"))
     }
 
+    @Test
+    fun `WHEN I submit data for multi pathogen organism THEN is accepted`() {
+        val accessionVersion = convenienceClient
+            .prepareDataTo(
+                status = Status.IN_PROCESSING,
+                organism = MULTI_PATHOGEN_ORGANISM,
+            )
+            .first()
+
+        submissionControllerClient.submitProcessedData(
+            PreparedProcessedData.successfullyProcessedMultiPathogenData(
+                accession = accessionVersion.accession,
+                version = accessionVersion.version,
+            ),
+            organism = MULTI_PATHOGEN_ORGANISM,
+        )
+            .andExpect(status().isNoContent)
+
+        convenienceClient.getSequenceEntry(
+            accession = accessionVersion.accession,
+            version = accessionVersion.version,
+            organism = MULTI_PATHOGEN_ORGANISM,
+        )
+            .assertStatusIs(Status.PROCESSED)
+    }
+
+    @Test
+    fun `WHEN I submit data for multi pathogen organism with main segment THEN is rejected`() {
```

Comment:
Unnecessary test, especially at this early stage in multi-pathogen feature work, it's already covered by normal tests that a segment needs to exist